java/java.lsp.server/src/org/netbeans/modules/java/lsp/server/protocol/TextDocumentServiceImpl.java

```diff
@@ -192,12 +205,14 @@
 
     private static final RequestProcessor BACKGROUND_TASKS = new RequestProcessor(TextDocumentServiceImpl.class.getName(), 1, false, false);
     private static final RequestProcessor WORKER = new RequestProcessor(TextDocumentServiceImpl.class.getName(), 1, false, false);
+    private static final Set<TextDocumentServiceImpl> ALL_INSTANCES = Collections.newSetFromMap(new WeakHashMap<>());
```

Comment:
What is `Set<TextDocumentServiceImpl> ALL_INSTANCES` good for? It is only used from `static void refreshAllEditors()` that seems to be unused.

Reply:
Sorry, this was part of a (failed) attempt to fix an unrelated problem and should not be here. Removed in 5987977.